src/utils/http.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+/**
+ * Base url for our api.
+ */
+const http = axios.create({
+  baseURL: 'https://hacker-news.firebaseio.com/v0'
```

Comment:
If possible use .env file and see https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables